cockatrice/src/userinfobox.cpp

```diff
@@ -131,7 +130,7 @@ void UserInfoBox::updateInfo(const ServerInfo_User &user)
         if (user.accountage_secs() == 0)
             accountAgeString = tr("Unknown");
         else {
-            qint64 seconds = user.accountage_secs();
+            auto seconds = static_cast<qint64>(user.accountage_secs());
```

Comment:
is this necessary?

Reply:
It was a warning that was simple enough to handle as it's an unsigned to signed conversion

Reply:
why not use unsigned then